pylint_odoo/checkers/itp_checks.py

```diff
@@ -132,6 +137,21 @@ def _check_manifest_image(self):
             else:
                 return False
 
+    @utils.check_messages('manifest-version')
+    def _check_manifest_version(self):
+        manifest_dict = self.manifest_dict
+        if "version" in manifest_dict.keys():
+            version = manifest_dict.get('version', '')
+            if version != '':
+                valid_odoo_versions = self.linter._all_options[
+                    'valid_odoo_versions'].config.valid_odoo_versions
+                if version[:4] != valid_odoo_versions[0].encode('ascii','ignore'):
```

Comment:
``'.'.join(version.split('.')[:2])`` is more future-proof aproach (imagine version 13.01 or 100.0, for 14.0-beta)

Reply:
awesome